Samples/custom-transform-adapter/config.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+)
+
+// Represents an adapter configuration (with routes, transforms, etc.)
+type Config struct {
+	D2CMessages []D2CMessage `json:"d2cMessages"`
+}
+
+type D2CMessage struct {
+	Path              string `json:"path"`              // Path filter for requests that will be routed to this transform
+	Transform         string `json:"transform"`         // jq query to tranform the request body
+	DeviceIdPathParam string `json:"deviceIdPathParam"` // Path parameter containing device Id
+	DeviceIdBodyQuery string `json:"deviceIdBodyQuery"` // jq query to pick the device Id from the request body
+	AuthHeader        string `json:"authHeader"`        // Header containing auth key
+	AuthQueryParam    string `json:"authQueryParam"`    // Query parameter containing auth key
+}
+
+// Loads, parses, and validates an adapter config from a file.
```

Comment:
golang doc comments are always supposed to start with the name of the thing they're documenting

```suggestion
// LoadConfig loads, parses, and validates an adapter config from a file.
```